app/build.gradle

```diff
@@ -67,19 +70,26 @@ android {
       dimension "version"
       versionNameSuffix "-internal"
     }
+    screenshots {
```

Comment:
I'm also going to add task which will copy fonts automatically from internal to screnshots

Reply:
Done